server/client_test.go

```diff
@@ -604,6 +604,7 @@ func TestAuthorizationTimeout(t *testing.T) {
 	if _, err := client.ReadString('\n'); err != nil {
 		t.Fatalf("Error receiving info from server: %v\n", err)
 	}
+	time.Sleep(secondsToDuration(serverOptions.AuthTimeout))
```

Comment:
Right, or, since we setup it up with options, you could do something like `serverOptions.AuthTimeout = 0.001` and do a `time.Sleep(100*time.Millisecond)` to avoid flapping. This is 100 times more than timeout and still low value so that we don't spend too much time in this test.